test/feeds.test.js

```diff
@@ -133,43 +133,58 @@ describe('test POST /feeds endpoint', () => {
   });
 });
 
-describe.skip('test DELETE /feeds/:id endpoint', () => {
+describe('test DELETE /feeds/:id endpoint', () => {
   let feedId;
-
-  beforeEach(async () => {
-    logout();
-    const user = login('Johannes Kepler', 'user1@example.com');
-    const feedData = {
+  function generateFeedData(user) {
+    return {
       author: user.name,
-      url: 'http://telescope200.cdot.systems',
+      url: `http://telescope200.cdot.systems/${Date.now()}`,
       user: user.id,
     };
-    const res = await request(app).post('/feeds').send(feedData).set('Accept', 'application/json');
-    expect(res.status).toEqual(201);
-    feedId = res.id;
-  });
-
-  afterAll(() => logout());
+  }
 
   it('should respond with a 403 status when not logged in', async () => {
+    // logging in user and generating a user
+    const user = login('Johannes Kepler', 'user1@example.com');
+    const feedData = generateFeedData(user);
+    let res = await request(app).post('/feeds').send(feedData).set('Accept', 'application/json');
+    expect(res.status).toEqual(201);
+    feedId = res.body.id;
     logout();
```

Comment:
Move this line down below your comment on line 155, since it's part of the logic of that block (i.e., you need to be logged out).